firebase-common/src/commonMain/kotlin/dev/gitlive/firebase/reencodeTransformation.kt

```diff
@@ -0,0 +1,22 @@
+package dev.gitlive.firebase
+
+import kotlinx.serialization.KSerializer
+
+inline fun <reified T> reencodeTransformation(value: Any?, builder: EncodeDecodeSettingsBuilder.() -> Unit = {}, transform: (T) -> T): Any? {
```

Comment:
houls these be public?

Reply:
Yes, just like encode/decode methods, they are to be used by the modules (in this case database) so a public modified is required to access them.